tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Nit: what is the relation of this root `tsconfig.json` with the addition of a root `eslintrc.cjs` ?

Reply:
the `.eslintrc.js` is a JS file which was being linted, but the linter complained that it couldn't find the TS project. This creates a minimal TS project that encompasses `.eslintrc.js`. I still hope to unify TypeScript config someday: https://github.com/Agoric/agoric-sdk/pull/4404